src/microbe_stage/editor/MutationPointsText.tres

```diff
@@ -0,0 +1,9 @@
+[gd_resource type="ShaderMaterial" load_steps=2 format=2]
```

Comment:
I think this file is a material? If so this should be named "MutationPointsTextMaterial.tres"